floris/tools/optimization/scipy/yaw.py

```diff
@@ -51,9 +55,26 @@ def __init__(
                 angle (deg). Defaults to 0.0.
             maximum_yaw_angle (float, optional): Maximum constraint on yaw
                 angle (deg). Defaults to 25.0.
-            x0 (iterable, optional): The initial yaw conditions (deg). If none
-                are specified, they are set to the current yaw angles for
-                all turbines. Defaults to None.
+            yaw_angles_baseline (iterable, optional): The baseline yaw angles
+                used to calculate the initial/baseline power production in the
+                wind farm and used to normalize the cost function. If the variable
+                exclude_downstream_turbines=True then the downstream turbines
+                that are excluded in the optimization will have their yaw angles
+                fixed to their value specified in this variable; yaw_angles_baseline.
```

Comment:
Update since this variable is no longer used when exclude_downstream_turbines = True (and for the other functions/classes where it is an argument too).